addons/languages/nias/pack/src/main/res/xml/nias_keyboards.xml

```diff
@@ -11,7 +11,7 @@
         nameResId="@string/nias_qwerty" />
     <Keyboard
         defaultDictionaryLocale="nia"
-        defaultEnabled="false"
+        defaultEnabled="true"
```

Comment:
Usually, only one layout will be default. If you have two, both will be enabled to new users which may give a weird user experience.